### ISSUE TITLE
Automatically and Regularly security scanning for container images

### ISSUE THREAD

**Issue by tenzen-y** [kubeflow/katib#2095]
/kind feature

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]
Currently, we manually scan container images to resolve security issues just before we release the new version of katib.

Although, this is inefficient, and we should scan container images regularly, not only just before releasing.

It might be better to run OSS tools for security scanning like the following on the GItHub Actions:

- [Snyk](https://github.com/snyk/cli)
- [`docker scan`](https://docs.docker.com/engine/scan/) (internally Snyk)
- [Trivy](https://github.com/aquasecurity/trivy)

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

---

Love this feature? Give it a 👍 We prioritize the features with the most 👍

**Comment by andreyvelich** [comment]
Thanks for creating this @tenzen-y!
Kubeflow community recently started discussion around security: https://github.com/kubeflow/kubeflow/issues/6662.
They are planing to establish Security WG in Kubeflow.

I think we should include this issue in the ROADMAP.

**Comment by tenzen-y** [comment]
@andreyvelich Thanks for letting me know!

> I think we should include this issue in the ROADMAP.

Does that mean we must work on this until tomorrow?
I may not have enough time today.

**Comment by andreyvelich** [comment]
>Does that mean we must work on this until tomorrow?

No, we don't have hard requirements to scan all images before the release.
We are planing to establish process after Kubeflow 1.7

**Comment by juliusvonkohout** [comment]
@difince wants to work on that as part of the security working group.

We already did scans for 1.7 and want to automate it for 1.9
